contentcuration/contentcuration/frontend/channelEdit/views/template/MoveModal.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <v-container grid-list-md>
+    <NewTopicModal
+      :showDialog="showNewTopicModal"
+      @createTopic="createTopic"
+      @cancelTopic="cancelTopic()"
+    />
+
+    <v-layout wrap>
+      <VDialog v-model="showDialog" fullscreen width="500" persistent>
+        <v-toolbar dark color="primary">
+          <VBtn icon dark @click="$emit('cancelMove')">
+            <VIcon>close</VIcon>
+          </VBtn>
+          <v-toolbar-title>
+            <b>{{ $tr("moveItems", {x: 2, title: currentNode.title}) }}</b>
+          </v-toolbar-title>
+          <v-spacer />
+        </v-toolbar>
+        <v-card>
+          <!-- header items -->
+          <v-layout wrap>
+            <v-flex md8 justify-start>
+              <v-breadcrumbs :items="crumbs" divider=">" />
+            </v-flex>
+            <v-spacer />
+            <v-flex shrink>
+              <VBtn flat @click="showTopicModal()">{{ $tr("addTopic") }}</VBtn>
+            </v-flex>
+          </v-layout>
+          <!-- list of children content -->
+          <v-card v-for="node in children" :key="node.id">
+            <v-layout class="card">
+              <v-flex xs2 align-self-center>
+                <v-img
+                  src="https://cdn.vuetifyjs.com/images/cards/foster.jpg"
```

Comment:
Ah I see- could you maybe add a comment here so we know to go back and add the thumbnail and/or card?